package vn.edu.huit.timetabling_gapo.repositories;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import vn.edu.huit.timetabling_gapo.entities.StudentGroup;

@Repository
public interface StudentGroupRepository extends JpaRepository<StudentGroup, String> {
}
